codecov.yml

```diff
@@ -0,0 +1,18 @@
+codecov:
+  notify:
+    require_ci_to_pass: yes
```

Comment:
Can we make this a non-blocking check?

Reply:
Yes. To [set non block status check](https://docs.codecov.io/docs/common-recipe-list#set-non-blocking-status-checks), make the project `informational: true` below.